pages/configuration/elasticsearch.rst

```diff
@@ -94,7 +94,7 @@ system defaults allow. **Set it to at least 64000 open file descriptors.**
 
 Graylog will show a notification in the web interface when there is a node in the Elasticsearch cluster which has a too low open file limit.
 
-Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/file-descriptors.html>`__ documentation pages.
+Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.7/file-descriptors.html>`__ documentation pages.
```

Comment:
theES 2.x link does not make sense - it must be 5.x for the name and the link needs to point to 5.6 what is the latest stable.

Reply:
@jalogisch there are more 2.x links in the Doc, should they all be moved to 6.x then? or 5.6?

Reply:
only 5.x and 6.x are supported. No need to link to unsupported versions documentation.

Reply:
To answer the question: I personal would move to 6.x if it makes sense.